The-Chonky-Panda/Atelier
# Eslint fix

## Description

ESLint is not signifying any linting errors so I made some minor change to the config file and fixed existing linting errors.

Also added an npm script for lint. Running `npm run lint` should now display any linting error in terminal.

## How to test

This PR is just fixing linting errors so it shouldn't cause any regression. Could pull this branch and verify locally that everything currently on dev should still render correctly.

## Notes


## Issue tracking

https://trello.com/c/qqN0jkqq
